modules/secure_storage_connector/main.tf

```diff
@@ -34,8 +35,9 @@ module "storage" {
   location            = data.azurerm_resource_group.group.location
   resource_group_name = data.azurerm_resource_group.group.name
   blob_container_name = var.namespace
-
   deletion_protection = var.deletion_protection
+  storage_key_id      = null
+  identity_ids        = ""
```

Comment:
It looks like these params were added here https://github.com/wandb/terraform-azurerm-wandb/pull/49/files

Reply:
It seemed like these fields are necessary for provisioning with a wb-managed key

Reply:
https://github.com/wandb/terraform-azurerm-wandb/pull/49/files#diff-1477e73501a9229a6501623ac074d620b1bdc7d056602a7e775f01e8c22e6c4dR33